cmd/api/main.go

```diff
@@ -121,6 +122,9 @@ func main() {
 		grpc.UnaryInterceptor(prometheus.UnaryServerInterceptor),
 	)
 	v1alpha2pb.RegisterResultsServer(s, v1a2)
+	if configFile.LOGS_API {
+		v1alpha2pb.RegisterLogsServer(s, v1a2)
+	}
```

Comment:
This makes sense to me - if we want logs to be an opt-in feature, disable the API completely if it is not available.